db/migrate/20180124130832_fix_devise_token_auth_issue.rb

```diff
@@ -0,0 +1,11 @@
+# https://github.com/lynndylanhurley/devise_token_auth/issues/121
+
+class FixDeviseTokenAuthIssue < ActiveRecord::Migration[5.1]
```

Comment:
Please give this migration a better name.

Reply:
Wasn't my migration lol